server/session.ts

```diff
@@ -0,0 +1,24 @@
+import { Express } from 'express';
+import session from 'express-session';
+
+import * as Config from './config';
+
+const SESSION_MAX_AGE_MILLIS = 60 * 60 * 1000;
+
+export default function setupSession(app: Express) {
+    app.set('trust proxy', 1);
+    app.use(
+        session({
+            cookie: {
+                maxAge: SESSION_MAX_AGE_MILLIS,
+                sameSite: 'lax',
```

Comment:
Tror denne skal kunne være strict nå?

Reply:
Muligens. Jeg har basert meg på https://github.com/navikt/security-blueprints/blob/master/examples/oidc-login-azuread/login-proxy-nodejs/src/session.js. Av https://github.com/navikt/pig/blob/master/sikkerhet/adr/000-samesite-openidconnect.md så kan det se ut som at `strict` kan funke for vårt case da, så jeg kan teste!

Reply:
Så ut til å funke fint det. Oppdaterte :)

Reply:
Viste seg å ikke funke mot AAD likevel, så jeg gikk tilbake til `lax` 😅